matplotcheck/tests/test_timeseries_module.py

```diff
@@ -1,9 +1,136 @@
-'''
-def test_assert_xydata_timeseries(pt_time_line_plt, pd_df_timeseries):
-    """Commenting this out for now as this requires a time series data object
-    this is failing because the time data needs to be in seconds like how
-    mpl saves it. """
-    pt_time_line_plt.assert_xydata(
-        pd_df_timeseries, xcol='time', ycol='A', xtime=True
+import pytest
+from matplotcheck.timeseries import TimeSeriesTester
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+
+@pytest.fixture
+def pd_df_timeseries():
+    """Create a pandas dataframe for testing, with timeseries in one column"""
+    return pd.DataFrame(
+        {
+            "time": pd.date_range(start="1/1/2018", periods=100),
+            "A": np.random.randint(0, 100, size=100),
+        }
     )
-'''
+
+
+@pytest.fixture
+def pt_time_line_plt(pd_df_timeseries):
+    """Create timeseries line plot for testing"""
+    fig, ax = plt.subplots()
+    pd_df_timeseries.plot("time", "A", kind="line", ax=ax)
+    axis = plt.gca()
+
+    return TimeSeriesTester(axis)
+
+
+"""
```

Comment:
I will merge this for now but i think we should pull this documentation out into an issue somewhere rather than leaving it in the code.